projects/ngrx.io/content/examples/store-elements/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,23 @@
+import { AppPage } from './app.po';
+import { browser, logging } from 'protractor';
+
```

Comment:
@timdeschryver should we remove all “.spec” files? I feel like they just add to the noise. These are defaults and do not actually test any of the counter logic.